docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,872 @@
+
+The following is a non-normative example of the PAR.
+
+.. code-block::
+
+    POST /as/par HTTP/1.1
+    Host: pid.it
+    Content-Type: application/x-www-form-urlencoded
+    
+    response_type=code
+    &client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+    &code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+    &code_challenge_method=S256
+    &request=eyJhbGciOiJSUzI1NiIsImtpZCI6ImsyYmRjIn0.ew0KIC Jpc3MiOiAiczZCaGRSa3F0MyIsDQogImF1ZCI6ICJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbSIsDQo gInJlc3BvbnNlX3R5cGUiOiAiY29kZSBpZF90b2tlbiIsDQogImNsaWVudF9pZCI6ICJzNkJoZFJrcXQz IiwNCiAicmVkaXJlY3RfdXJpIjogImh0dHBzOi8vY2xpZW50LmV4YW1...
+    &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation
+    &client_assertion=$WalletInstanceAttestation$ 
+
+The JWS header of request object is represented below:
+
+.. code-block:: JSON
+
+  {
+    "alg": "ES256",
+    "kid": "FifYx03bnosD8m6gYQIfNHNP9cM_Sam9Tc5nLloIIrc",
+  }
+
+
+The JWS payload of the request object is represented below:
+
+.. code-block:: JSON
+
+    {
+    "response_type":"code",
+    "client_id":"$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$",
+    "state":"fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd",
+    "code_challenge":"E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM",
+    "code_challenge_method":"S256",
+    "authorization_details":[
+    {
+        "type":"openid_credential",
+        "format": "vc+sd-jwt",
+        "credential_definition": {
+            "type": ["eu.eudiw.pid.it"]
+        }
+    }
+    ],
+    "redirect_uri":"eudiw://start.wallet.example.org",
+    "client_assertion_type":"urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation",
+
+**Step 7:** The PID Provider creates a new request URI representing this new authorization request to be returned to the Wallet Instance. 
+    }
+
+
+.. note::
+    **Federation Check:** PID Provider needs to check that the Wallet Provider is part of the federation and in addition it MUST verify Wallet Instance Attestation validity by checking its signature and the validity of the claims inside it. 
+
+**Step 7:** The PID Provider creates a new request URI representing this new authorization request and returns it to the Wallet Instance. 
+
+.. code-block:: 
+
+    HTTP/1.1 201 Created
+    Content-Type: application/json
+    Cache-Control: no-cache, no-store
+    
+    {
+        "request_uri":"urn:ietf:params:oauth:request_uri:bwc4JK-ESC0w8acc191e-Y1LTC2",
+        "expires_in": 60
+    }
+    Host: pid-provider.example.org
```

Comment:
please bring this up, and also is Cache-Control required in this request? If not please remove it.